src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -169,7 +169,7 @@ export default function ChartExpandedState({ asset }) {
             chainId: asset.chainId,
             network: asset.network,
             address: asset.address,
-            mainnetAddress: asset?.networks?.[useBackendNetworksStore.getState().getChainsName()[ChainId.mainnet]]?.address,
+            mainnetAddress: asset?.networks?.[ChainId.mainnet]?.address,
```

Comment:
if asset.networks is now mapped by chainId, are there other areas we need to consider this change?

Reply:
oh it's already mapped by chain Id, it was just incorrectly using chains name here!